### PR TITLE
xbill: init at 2.1

### PR DESCRIPTION
**Pull request by herrwiese**
Old game, once more popular than Quake (but not Quake Ⅱ).

From the homepage:

Ever get the feeling that nothing is going right? You're a sysadmin, and
someone's trying to destroy your computers. The little people running
around the screen are trying to infect your computers with Wingdows
[TM], a virus cleverly designed to resemble a popular operating system.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by herrwiese** [comment]
@GrahamcOfBorg build pkgs.xbill

File: pkgs/games/xbill/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchurl, libX11, libXpm, libXt, motif, ... }:
+
+stdenv.mkDerivation rec {
+  name = "xbill-${version}";
```
Comment (Lassulus) [inline pkgs/games/xbill/default.nix]:
```suggestion
  pname = "xbill";
```

File: pkgs/games/xbill/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  ];
+
+  src = fetchurl {
+    url = "http://www.xbill.org/download/xbill-2.1.tar.gz";
+    sha256 = "13b08lli2gvppmvyhy0xs8cbjbkvrn4b87302mx0pxrdrvqzzz8f";
+  };
+
+  meta = with stdenv; {
+    description = "Protect a computer network from getting infected.";
+    homepage = http://www.xbill.org/;
```
Comment (Lassulus) [inline pkgs/games/xbill/default.nix]:
```suggestion
    homepage = "http://www.xbill.org/";
```

**Review by Lassulus — CHANGES_REQUESTED** [review]

File: pkgs/games/xbill/default.nix (hunk 1 above)
Comment (herrwiese) [reply pkgs/games/xbill/default.nix]: Thanks, I addressed those.

**Comment by Lassulus** [comment]
tested with nix-review, played a couple of levels. I couln't warp to other levels, but ¯\_(ツ)_/¯

**Review by Lassulus — APPROVED** [review]

**Pull request merged**